components/persistor/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@haventech/persistor",
     "description": "A subclass of supertype that serializes to and reconstitutes from MongoDB or SQL databases",
     "homepage": "https://github.com/haven-life/persistor",
-    "version": "12.0.0",
+    "version": "12.0.1",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
```

Comment:
Looks like we have a test case failure (pipeline seems to be failing)

can we make sure to add unit tests around: 

1. no warning message is created if we are deleting an index that is already deleted
2. no warning message is created if we insert an index that was already created
3. If a new index is added in schema file, then pg_indexes has that index.
4. If that same index is deleted from schema file, pg_indexes removes it without impacting any other index

Reply:
@kamranHaven  I have covered all the scenarios you mention in the comments. 

About the regression:

1. I have tested with blank db -> works as expected. We need to double the indexes created by the amorphic one. 
2.  Not sure how I should test it. I'll check with the PA repo if there are any changes in the schema/indexes/columns and maybe try to run a startup.